src/Telemetry/Library/TelemetryManagerConfig.cs

```diff
@@ -39,13 +55,30 @@ public string EventNamePrefix
             set { this.eventNamePrefix = ValidateName(value); }
         }
 
+        /// <summary>
+        /// The default User Consent to collect telemetry.
+        /// If the default value is ConsentKind.OptedOut, then the user can opt-in via
+        /// the environment variable defined in TelemetryOptInVariableName.
+        /// If the default value is ConsentKind.OptedIn, then the user can opt-out via
+        /// the environment variable defined in TelemetryOptOutVariableName.
+        /// </summary>
+        public ConsentKind DefaultTelemetryConsent { get; set; } = ConsentKind.OptedOut;
+
         /// <summary>
         /// The name of the environment variable to be checked at initialization to see if
         /// the user wants to opt-out of sending telemetry data.
         /// If the value of the environment variable is "1", no data will be collected or sent.
         /// </summary>
         public string TelemetryOptOutVariableName { get; set; } = "QDK_TELEMETRY_OPT_OUT";
 
+        /// <summary>
+        /// The name of the environment variable to be checked at initialization to see if
+        /// the user wants to opt-in of sending telemetry data.
+        /// If the value of the environment variable is "1", data will be collected or sent.
+        /// The "TelemetryOptOut" will always take precedence.
+        /// </summary>
+        public string TelemetryOptInVariableName { get; set; } = "QDK_TELEMETRY_OPT_IN";
```

Comment:
1. I have a suspicion that "TelemetryOptOut" did not survice (mentioned in the fragment "/// The "TelemetryOptOut" will always take precedence.").
2. 
```suggestion
        /// <summary>
        /// The name of the environment variable to be checked at initialization to see if
        /// the user wants to opt-in of sending telemetry data.
        /// If the value of the environment variable is "1", data will be collected and sent.
        /// The "TelemetryOptOut" will always take precedence.
        /// </summary>
        public string TelemetryOptInVariableName { get; set; } = "QDK_TELEMETRY_OPT_IN";
```